packages/dialtone-vue2/recipes/conversation_view/message_input/message_input.vue

```diff
@@ -775,15 +784,24 @@ export default {
   border-radius: var(--dt-size-radius-400);
   border: var(--dt-size-border-100) solid;
   border-color: var(--dt-color-border-default);
+  box-shadow: 0 0 0 0 rgba(0, 0, 0, 0%);
+  line-height: var(--dt-font-line-height-400);
   cursor: text;
+  transition: all 100ms cubic-bezier(0.22, 1, 0.36, 1);
 
   &:focus-within {
     border-color: var(--dt-color-border-bold);
-    box-shadow: var(--dt-shadow-small);
+    box-shadow: 0 0 var(--dt-size-300) 0 var(--dt-color-surface-moderate-opaque);
+    transition: all 250ms cubic-bezier(0.22, 1, 0.36, 1);
+  }
+
+  &:not(:hover):not(:focus-within) .dt-message-input__button {
+    opacity: 0.75;
+    transition: all 250ms cubic-bezier(0.22, 1, 0.36, 1);
```

Comment:
Can all this transitions be unified into one? or is it necessary to have 3 of them?

Reply:
I swapped these transition properties out for Dialtone variables.